app/controllers/idv/otp_delivery_method_controller.rb

```diff
@@ -25,6 +25,23 @@ def create
 
     private
 
+    attr_reader :idv_phone
+
+    def view_locals
+      {
+        gpo_letter_available: gpo_letter_available,
+        phone_number_capabilities: phone_number_capabilities,
+      }
+    end
+
+    def phone_number_capabilities
+      PhoneNumberCapabilities.new(idv_phone, phone_confirmed: user_phone?)
+    end
+
+    def user_phone?
+      MfaContext.new(current_user).phone_configurations.any? { |config| config.phone == idv_phone }
```

Comment:
do we want to make sure `idv_phone` and `config.phone` are both formatted/normalized when we compare?

Reply:
```suggestion
      MfaContext.new(current_user).phone_configurations.any? { |config| config.formatted_phone == Phonelib.parse(idv_phone).international }
```

Reply:
This method was pulled (mostly) verbatim from another class, FWIW.